pkg/tests/tasks/traffic/circuit_breaking_test.go

```diff
@@ -51,10 +51,11 @@ func TestCircuitBreaking(t *testing.T) {
 
 		t.LogStep("Verify connection with curl: expect 200 OK")
 		retry.UntilSuccess(t, func(t test.TestHelper) {
+			httpbinIP := oc.GetPodIP(t, pod.MatchingSelector("app=httpbin", ns))
```

Comment:
Better to use the Service IP instead of the Pod IP here, just to keep things as they were.

Reply:
Done